### PR TITLE
install_resolver_dependencies keys are printed in the tool_list.yml

### PR DESCRIPTION
**Pull request by drosofff**
with a nasty trick to preserve a more readable order of keys in the yml file (name first)
install_resolver_dependencies is always set to True

### CODE REVIEW

File: ephemeris/generate_tool_list_from_ga_workflow_files.py [hunk 1]
```diff
@@ -49,14 +48,24 @@ def translate_workflow_dictionary_to_tool_list(tool_dictionary, panel_label):
     tool_list = []
     for tool in starting_tool_list:
         sub_dic = {'name': tool['name'], 'owner': tool['owner'], 'revision': tool['changeset_revision'],
-                   'tool_panel_section_label': panel_label, 'tool_shed_url': 'https://'+tool['tool_shed']}
+                  'tool_panel_section_label': panel_label, 'tool_shed_url': 'https://'+tool['tool_shed'],
+                  'zinstall_resolver_dependencies': 'True'}
```
Comment (bgruening) [inline ephemeris/generate_tool_list_from_ga_workflow_files.py]: the `z` looks strange :)

Comment (drosofff) [inline ephemeris/generate_tool_list_from_ga_workflow_files.py]: Yes this is the nasty trick I mentionned... This is not a bug, this is a feature :-/

**Comment by bgruening** [comment]
@drosofff do you mind to add a test for this tool?

File: ephemeris/generate_tool_list_from_ga_workflow_files.py (hunk 1 above)
Comment (drosofff) [inline ephemeris/generate_tool_list_from_ga_workflow_files.py]: see line 63

**Comment by drosofff** [comment]
@bgruening No I don't. I put it in my todo list !

**Comment by drosofff** [comment]
@bgruening here is a test. Works on my fork, should work on the upstream.

**Comment by bgruening** [comment]
@drosofff thanks for the tests it looks good.
I have a general comment/hint that hopefully makes this patch easier. Have you seen this patch? https://github.com/galaxyproject/ephemeris/pull/26

With this in place you only need to specify it once on top of all tools and don't need to mess with he ordering :)

**Comment by bgruening** [comment]
Thanks @drosofff!

**Pull request merged**